packages/eslint-plugin/src/rules/no-dynamic-delete.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  TSESTree,
+  AST_NODE_TYPES,
+} from '@typescript-eslint/experimental-utils';
+import * as tsutils from 'tsutils';
+import * as util from '../util';
+import { ReportFixFunction } from '@typescript-eslint/experimental-utils/dist/ts-eslint';
```

Comment:
nit - don't import the `dist` folder - this exact location is an implementation detail.
It should be available in the TSESLint namespace instead.